module/zfs/vdev.c

```diff
@@ -1858,7 +1858,7 @@ vdev_open(vdev_t *vd)
 	 */
```

Comment:
While you're here, this comment (with author's initials) should probably be removed or rewritten.

Reply:
Done.  I opted to remove the comment.